aws/cloudtrail/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "s3_bucket_iam_user_name" {
```

Comment:
Would it make sense to add the access key definition in this module, perhaps as an optional feature?

Reply:
That's a good question... I'm not sure. We either need to have this module create it and output those credentials, or have this module output the name of the created user, so calling code can create an access key.

I suppose it does seem cleaner to have the module that creates the user take care of (optionally) creating the access key. I'll try that out, and see how cleanly it goes.